app/src/main/java/com/uc/mymovies/main/MainFragment.kt

```diff
@@ -0,0 +1,13 @@
+package com.uc.mymovies.main
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+
+class MainFragment : Fragment() {
+    private lateinit var viewModel: MainViewModel
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
+    }
+}
```

Comment:
create fragment for layout

Reply:
Does the new fragment have a purpose?  Does it have a layout?  If so, I don't see it.

Is it decreasing technical debt?